### PR TITLE
Upgrade @looker/extension-sdk and @looker/components

### PR DESCRIPTION
**Pull request by bryans99**
Upgrade of @looker/components requires minor changes to the extension. As prettier is turned on there is some reformatting. In addition, the logout button was moved so that it consistent with other buttons on that view.

### CODE REVIEW

File: src/components/ExternalApiFunctions/ExternalApiFunctions.tsx [hunk 1]
```diff
@@ -77,38 +92,44 @@ export const ExternalApiFunctions: React.FC<ExternalApiFunctionsProps> = () => {
...
-        <Divider/>
-        <Auth dataDispatch={dataDispatch} dataState={dataState}/>
-        <Divider/>
-        {jwtToken &&
+        <Divider />
+        <Auth dataDispatch={dataDispatch} dataState={dataState} />
+        <Divider />
+        {jwtToken && (
           <>
             <TabList selectedIndex={selectedIndex} onSelectTab={onSelectTab}>
```
Comment (josephaxisa) [inline src/components/ExternalApiFunctions/ExternalApiFunctions.tsx:112]: Not related to this PR but fwiw, the `useTabs` hook is pretty neat when working with `TabList` and `TabPanel`.

File: src/App.tsx [hunk 2]
```diff
@@ -64,8 +64,7 @@ export const App: React.FC<AppProps> = hot(() => {
       requiredLookerVersion=">=7.9.0"
     >
       <ThemeProvider theme={theme}>
-        <>
-          <GlobalStyle />
+        <ComponentsProvider>
```
Comment (josephaxisa) [inline src/App.tsx:67]: `ComponentsProvider` is essentially a`ThemeProvider` wrapper so `ThemeProvider` is not needed here.

Comment (bryans99) [reply src/App.tsx:67]: Good catch! Thanks

File: src/components/ExternalApiFunctions/ExternalApiFunctions.tsx (hunk 1 above)
Comment (bryans99) [reply src/components/ExternalApiFunctions/ExternalApiFunctions.tsx:112]: Yeah, that wasn't available (or was not documented) when I wrote this. Not going to bother refactoring

File: src/App.tsx (hunk 2 above)
Comment (jkaster) [reply src/App.tsx]: Hah. I missed that lingering `ThemeProvider`. Good catch.

**Pull request merged**